package.nls.json

```diff
@@ -451,6 +451,8 @@
     "DataScience.unhandledMessage": "Unhandled kernel message from a widget: {0} : {1}",
     "DataScience.qgridWidgetScriptVersionCompatibilityWarning": "Unable to load a compatible version of the widget 'qgrid'. Consider downgrading to version 1.1.1.",
     "DataScience.kernelStarted": "Started kernel {0}",
+    "DataScience.kernelStarting": "_Connecting to kernel..._",
```

Comment:
I don't know if translators for this will understand they have to keep the ```_``` on either side. I think the ```_``` should probably be in code.